### PR TITLE
SET-317: implement commit retrieval

### PR DESCRIPTION
**Pull request by michpetrov**
Issue: [SET-317](https://projects.engineering.redhat.com/browse/SET-317)

Adding a method to retrieve a list of commits from a repo, needed to check if a PR has been merged into a given branch.

### CODE REVIEW

**Review by baranowb — CHANGES_REQUESTED** [review]

File: domain/src/main/java/org/jboss/set/aphrodite/domain/Commit.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
+/*
```
[the remark this answers is not in the document]
Comment (michpetrov) [reply domain/src/main/java/org/jboss/set/aphrodite/domain/Commit.java:1]: PullRequest keeps a List<String> of commits, do you want to change that to Commit? I haven't found other uses.

**Comment by michpetrov** [comment]
@baranowb waiting on your feedback.

**Comment by baranowb** [comment]
@michpetrov  and you shall receive it!

File: domain/src/main/java/org/jboss/set/aphrodite/domain/Commit.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (baranowb) [reply domain/src/main/java/org/jboss/set/aphrodite/domain/Commit.java:1]: yes, either way. It needs to be the same in both, either one version or another.

**Comment by michpetrov** [comment]
@baranowb updated and rebased

**Pull request merged**